src/models/TestResultsDAO.ts

```diff
@@ -68,7 +68,7 @@ export class TestResultsDAO {
     const { testerStaffId } = filters;
     const keyCondition =
       "testerStaffId = :testerStaffId AND testStartTimestamp > :testStartTimestamp";
-    let filterExpression = "testEndTimestamp < :testEndTimestamp";
+    let filterExpression = "testEndTimestamp <= :testEndTimestamp";
```

Comment:
This change has already been merged as part of another ticket, please rebase

Reply:
It has now been successfully rebased 👍🏼